File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice03.py
def algorithme1():
    N1=float(input("saisir un reel N1: "))
    N2=float(input("saisir un reel N2: "))
    if((N1>0 and N2>0 ) or (N1<0 and N2<0)):
        print("le produit et positif.")
    else:
        print("le produit et négatif.")
def algorithme2():
    N1=float(input("saisir un reel N1: "))
    N2=float(input("saisir un reel N2: "))
    if(N1==0 or N2==0):
        print("le produit et nul.")
    elif((N1>0 and N2>0 ) or (N1<0 and N2<0)):
        print("le produit et positif.")
    else:
        print("le produit et négatif.")
algorithme1()
algorithme2()

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice03.c
#include <stdio.h>
float N1,N2;
int algorithme1(){
    printf("saisir deux nombre N1 et N2: \n");
    scanf("%f %f",&N1,&N2);
    if((N1>0 && N2>0) || (N1<0 && N2<0))
        printf("le produit et positif. \n");
    else
        printf("le produit est négatif.\n");
    return 0;
}
int algorithme2(){
    printf("saisir deux nombre N1 et N2: \n");
    scanf("%f %f",&N1,&N2);
    if((N1>0 && N2>0) || (N1<0 && N2<0))
        printf("le produit et positif.\n");
    else if((N1==0) || (N2==0))
        printf("le produit est nul\n.");
    else
        printf("le produit est négatif.\n");
    return 0;
}
int main(){
    algorithme1();
    algorithme2();
}